packages/stackable/stackables/base.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+class BaseStackable {
```

Comment:
I really think this should live inside the "config" module, as it would be depended on by the config, and we must use the config everywhere anyway.